tasks/components/watchdog/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+# Check if the variable is not blank
+if [[ ! -z "$CONFIG_DS_WATCHDOG_DEVICE" ]]; then
+    # Write the configuration to /etc/watchdog.conf
+    cat > /etc/watchdog.conf << EOF
+watchdog-device = $CONFIG_DS_WATCHDOG_DEVICE
+watchdog-timeout = $CONFIG_DS_WATCHDOG_TIMEOUT
+realtime = yes
+priority = 1
```

Comment:
I believe 1 is the lowest real-time priority? I think we'd want the highest if we're going to set this as a real-time task.

Reply:
Per discussion outside of git, these are the Debian defaults for realtime and prio.